trajectorytools/trajectories.py

```diff
@@ -272,6 +272,28 @@ def load(cls, filename):
         loaded_dict = np.load(filename, allow_pickle=True).item()
         return cls(loaded_dict["traj_data"], loaded_dict["params"])
 
+    def export_trajectories_to_csv(self, csvfilename, delimiter=",", **kwargs):
+        """Export trajectories to a csv plain text file.
+
+        :param csvfilename: Name of the csv file to be created
+        :param delimiter: Delimiter (default is ",")
+        :param kwargs: Other keyword arguments for numpy.savetxt
+        """
+        data_to_save = np.concatenate([self.s, self.v, self.a], axis=-1)
+        data_to_save = data_to_save.reshape(self.number_of_frames, -1)
+        data_header = ["x", "y", "vx", "vy", "ax", "ay"]
+        individuals = range(self.number_of_individuals)
+        header = delimiter.join(
+            [f"{h}_{i}" for i in individuals for h in data_header]
+        )
+        np.savetxt(
```

Comment:
Does this test uses interpolate=True? In case it does, how are NaN saved, and how are they read when retrieved? 
Maybe add a comment in docstring about missing values (NaNs).

Reply:
This is handled by np.savetxt. Unsure how to document

No, I do not test the case with nans, I could do.